R/axis-secondary.R

```diff
@@ -187,21 +187,49 @@ AxisSecondary <- ggproto("AxisSecondary", NULL,
 
     # patch for date and datetime scales just to maintain functionality
     # works only for linear secondary transforms that respect the time or date transform
-    if (scale$trans$name %in% c("date", "time")){
+    if (scale$trans$name %in% c("date", "time")) {
```

Comment:
@thomasp85 can you please review this part of the PR?

Reply:
LGTM...(I also think we have gotten the secondary axis functionality pretty well-covered by unit tests by now)